src/app/containers/ArticleMain/index.jsx

```diff
@@ -21,26 +35,53 @@ const componentsToRender = {
   timestamp,
 };
 
-const ArticleMain = ({ articleData }) => {
-  const { content, metadata, promo } = articleData;
-  const { blocks } = content.model;
+const ArticleMain = ({ articleData: data }) => {
+  const {
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+    articleAuthor,
+  } = useContext(ServiceContext);
+
+  const headline = getHeadline(data);
+  const description = getSummary(data) || getHeadline(data);
+  const linkedData = buildLinkedData({
+    ...data,
+    brandName,
+    noBylinesPolicy,
+    defaultImage,
+  });
 
   return (
     <>
-      <ATIAnalytics data={articleData} />
-      <ChartbeatAnalytics data={articleData} />
-      <MetadataContainer metadata={metadata} promo={promo} />
+      <ATIAnalytics data={data} />
+      <ChartbeatAnalytics data={data} />
+      <ArticleMetadata
```

Comment:
IMO `ArticleMetadata` should be a container, with alot of the functions this file calls, being inside it. It also resolves the oddity of a component calling a container

Reply:
Have moved `ArticleMetadata` into containers